test/data/javascript/tests.js

```diff
@@ -1708,6 +1708,30 @@ exports.test_data = {
         {
           input: 'if (a)\n{\nb;\n}\nelse\n{\nc;\n}',
           output: 'if (a) {\n    b;\n} else {\n    c;\n}'
+        },
+        {
+          unchanged: 'fn`tagged`'
+        },
+        {
+          unchanged: 'fn()`tagged`'
+        },
+        {
+          unchanged: 'fn`${algo} ${`6string`}`'
+        },
+        {
+          unchanged: 'fn`${fn2()} more text ${`${`more text`}`} banana ${fn3`test`} ${fn4()`moretest banana2`}`'
+        },
+        {
+          input: '`untagged`+`untagged`',
+          output: '`untagged` + `untagged`'
+        },
+        {
+          input: "fun() `taggedd`",
+          output: "fun()\n`taggedd`"
```

Comment:
```suggestion
          output: "fun() `taggedd`"
```

Reply:
So this should be unchanged then?

Reply:
@JuanFML
Unchanged.  If they put a space in there, it isn't valid syntax but this tool should not assume it knows what they meant.

Reply:
I have added the changes